### PR TITLE
[Fix] ParlerTTSForCausalLM.generate discards the prepared attention mask

### PR DESCRIPTION
**Pull request by yushuosun**
## Motivation

`ParlerTTSForCausalLM.generate` silently runs without the attention mask it computes, which can degrade generation (especially with padding) and uses a deprecated pre-4.46 call signature.

## Root cause

In `parler_tts/modeling_parler_tts.py`, `ParlerTTSForCausalLM.generate` does:

```python
if model_kwargs.get("attention_mask", None) is None and requires_attention_mask:
    self._prepare_attention_mask_for_generation(
        input_ids, generation_config.pad_token_id, generation_config.eos_token_id
    )
```

The return value is discarded (never assigned to `model_kwargs["attention_mask"]`), so the computed mask is thrown away. It also passes the pre-4.46 positional `pad_token_id`/`eos_token_id`. The sibling `ParlerTTSForConditionalGeneration.generate` in the same file does it correctly:

```python
model_kwargs["attention_mask"] = self._prepare_attention_mask_for_generation(
    inputs_tensor, generation_config._pad_token_tensor, generation_config._eos_token_tensor
)
```

## Modifications

`parler_tts/modeling_parler_tts.py` (`ParlerTTSForCausalLM.generate`): assign the prepared mask back to `model_kwargs["attention_mask"]` and pass `generation_config._pad_token_tensor`/`_eos_token_tensor`, mirroring the conditional-generation sibling. (`_prepare_special_tokens` is already called just above, so the `_*_tensor` attributes are populated.)

## Duplicate-check

- Track A — `gh api 'repos/huggingface/parler-tts/pulls?state=open&per_page=100' --paginate` grepped for `attention_mask` / `_prepare_attention` → no open PR touches this block (the PRs touching this file edit cache handling elsewhere).
- Track B — no open issue/comment claims this fix.

### CODE REVIEW

**Comment by yushuosun** [comment]
Closing after prolonged project inactivity and no maintainer response. The branch remains available if maintenance resumes.
